packages/sources/therundown/test/integration/schedule.ts

```
@@ -1,18 +1,45 @@
 import { assertError, assertSuccess } from '@chainlink/ea-test-helpers'
 import { AdapterRequest } from '@chainlink/types'
 import request from 'supertest'
+import type { SuperTest, Test } from 'supertest'
 import {
-  mockScheduleResponseError1,
-  mockScheduleResponseMalformed1,
+  mockScheduleResponseError,
+  mockScheduleResponseMalformedMarketCreate,
+  mockScheduleResponseMalformedMarketResolve,
   mockScheduleResponseSuccess,
 } from './fixtures'
 
 export function scheduleTests(): void {
+  const req: SuperTest<Test> = request('localhost:8080')
   const id = '1'
-  const req = request('localhost:8080')
 
   describe('error calls', () => {
-    describe('when unsuccessfully requesting Therundown API', () => {
+    describe('when unsuccessfully requesting Therundown API with Market Create', () => {
```

The tests below are testing the same thing:
- `error calls when unsuccessfully requesting Therundown API with Market Create should throw an exception`
- `error calls when unsuccessfully requesting Therundown API with Market Resolve should throw an exception`

The response from `mockScheduleResponseError` fixture will throw an error in `requester.ts`. It is not input-parameter dependent. Remove one of the tests and amend the test removing the market mention, e.g:
`error calls when unsuccessfully requesting Therundown API should throw an exception`

---

Resolved